invenio_vocabularies/datastreams/writers.py

```diff
@@ -108,3 +108,16 @@ def write(self, stream_entry, *args, **kwargs):
             yaml.safe_dump([stream_entry.entry], file)
 
         return stream_entry
+
+
+class PrintWriter(BaseWriter):
```

Comment:
major: was this added for testing resons? If yes, I think we should remove it (also from the config above). Otherwise, it should print on the console (currently saving to a yaml file 🤔). This brings me to:

If we are going to print, wouldnt it be better to implement a logging write that is flexible enough? (there was an issue about it but I cant find it).

My take: remove `PrintWriter` seems only used for tests/ad-hoc.

Reply:
removed